planetmint/backend/tarantool/query.py

```diff
@@ -168,10 +168,13 @@ def get_asset(connection, asset_id: str):
 def get_assets(connection, assets_ids: list) -> list:
     _returned_data = []
     for _id in list(set(assets_ids)):
-        asset = get_asset(connection, _id)
-        _returned_data.append(asset)
+        res = connection.run(
+            connection.space("assets").select(_id, index="txid_search")
```

Comment:
I can do that. Did it this way because get_asset does not return the full asset entry and therefor we had problems with the sorting. However, I can check where and how the get_asset method is used and what might be affected if i change the return value